dev_playbook.yml

```diff
@@ -122,3 +126,11 @@
       with_items:
         - "\t\tcp {{ nginx_log_olddir_root }}/* {{ nginx_long_term_log_dir }}/"
 
+    - name: Copy galaxy_jwd.py script to walle_script_location
```

Comment:
why do this in dev playbook and not other playbooks?

Reply:
Good catch - this is now included in walle role.